docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+    node:
+      build: './stackle_api'
+      ports:
+        - "8080:8080"
+      links :
+        - mongo
+    mongo:
+      image : mongo
+      ports:
+        - "27017:27017"
```

Comment:
If you don't need to open the port `27017` to the host, you can omit this part, `links` to `mongo` will make `27017`  port accessible to `node` service I guess. Check that too.

Reply:
changed it in the new commit. Works fine 👍